src/lib.rs

```diff
@@ -1,8 +1,18 @@
-#![cfg_attr(test, deny(warnings))]
+#![cfg(test)]
 #![warn(rust_2018_idioms)]
 
 mod cnab;
 pub use crate::cnab::*;
 
+#[macro_use]
+extern crate serde;
+#[macro_use]
+extern crate serde_json;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate canonical_json;
+extern crate spectral;
```

Comment:
This crate is using Rust 2018 which means no need for the `extern crate` declarations.